tests/test_config_utils.py

```diff
@@ -9,6 +10,12 @@
 ORIGINAL_CONFIG = copy.deepcopy(bigchaindb._config)
 
 
+@pytest.fixture
+def ignore_local_config_file(monkeypatch):
+    from bigchaindb.config_utils import file_config
+    monkeypatch.setattr('bigchaindb.config_utils.file_config', file_config)
```

Comment:
This is for testing `file_config`. I have not found another way to disable the "autoused" fixture: https://github.com/bigchaindb/bigchaindb/blob/master/tests/conftest.py#L32-L39

Another approach would be to not use `autouse` and to explicitly use the mocked `file_config` when needed.

I consider this to be a temporary approach, and would like to address the issue more radically via the pytest-bigchaindb plugin (#462(.